fix(rbd): enc at rest (#3516)
scaleway/docs-content: managed-databases/postgresql-and-mysql/how-to/create-a-database.mdx

        <Message type="important">
          You can upgrade a standalone node to High Availability after Database Instance creation, but not the opposite.
        </Message>
    - Enable encryption at rest if you want to ensure the permanent encryption of your database data.
        <Message type="important">

>           When you enable encryption at rest, you cannot disable it after database creation. All databases, all data and all snapshots will be encrypted. Backup encryption is currently not available.

Reviewer: same as before

Author: ```suggestion
          When you enable encryption at rest, you cannot disable it after database creation. All databases, all data (including logs) and all snapshots will be encrypted. Logical backup encryption is currently not available.
```

>           When you enable encryption at rest, you cannot disable it after database creation. All databases, all data (including logs) and all snapshots will be encrypted. Logical backup encryption is currently not available.

Reviewer 2: ```suggestion
          When you enable encryption at rest, you cannot disable it after database creation. All databases, data (including logs), and snapshots will be encrypted. Logical backup encryption is not currently available.
```

          <br /><br />

>           During the beta phase, if chose to enable encryption after database creation, you must make an enablement request via a [support ticket](https://console.scaleway.com/support).

Reviewer: same as before

Author: ```suggestion
          During the beta phase, you can enable encryption on an existing Database Instance by opening a request via a [support ticket](https://console.scaleway.com/support).
```

          <br /><br />

>           Enabling encryption incurs in the copy of the database data into a new, encrypted block volume. This takes approximately 1h per 100GB of storage. A couple seconds of downtime are also expected towards the end of the copying process, when the volumes are being switched.

Reviewer 2: ```suggestion
          Enabling encryption means your database data will be copied into a new, encrypted block volume. This takes approximately 1 hour per 100 GB of storage. When switching volumes, expect a few seconds of downtime towards the end of the copying process.
```
